src/Commands/Sql/SqlDatabaseListArguments.cs

```diff
@@ -0,0 +1,7 @@
+namespace AzureMcp.Commands.Sql;
+
+public sealed class SqlDatabaseListArguments(string? Subscription, string? ServerName)
+{
+    public string? Subscription { get; } = Subscription;
+    public string? ServerName { get; } = ServerName;
+}
```

Comment:
Please put the arguments into src/Arguments/{Service}/{SubService}/{Resource}/{Resource}{Operation}Arguments.cs